src/main/java/salesforce/restclient/RestApi.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package salesforce.restclient;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+
+import salesforce.utils.AccessToken;
+import salesforce.utils.TokenConstant;
+
```

Comment:
imports are not necessary separate

please press ctrl + alt + O to the identation of the comments